opensrp-pnc/src/main/java/org/smartregister/pnc/repository/PncStillBornRepository.java

```diff
@@ -55,4 +58,17 @@ public boolean delete(PncStillBorn stillBorn) {
         throw new NotImplementedException("");
     }
 
+    public int count(@NonNull String baseEntityId) {
+        int count = 0;
+        SQLiteDatabase sqLiteDatabase = getReadableDatabase();
```

Comment:
Can the method name be descriptive ... and is this method necessary

Reply:
method renamed.